src/domain/products/components/sales-channels-modal/index.tsx

```diff
@@ -6,7 +6,6 @@ import LayeredModal, {
   LayeredModalContext,
 } from "../../../../components/molecules/modal/layered-modal"
 import AvailableScreen from "./available-screen"
-import { SalesChannelsModalContext } from "./use-sales-channels-modal"
```

Comment:
todo: This seems to be breaking the site and should not be removed

Reply:
Ooops, I don't know how that happened. It should be good now!